pkg/storage/secret/metadata/data/keeper_listByName.sql

```diff
@@ -17,7 +17,6 @@ FROM
   {{ .Ident "secret_keeper" }}
 WHERE 1 = 1 AND
   {{ .Ident "namespace" }} = {{ .Arg .Namespace }} AND
-  {{ .Ident "type" }} != {{ .Arg .ExcludeSQLKeeper }} AND
```

Comment:
This type does not exist anymore, so we don't need to restrict/special-case this query